Policies/github-terms-of-service.md

```diff
@@ -136,25 +159,33 @@ Some Accounts may have private repositories, which allow the User to control acc
 
 #### 3. Access
-GitHub employees may only access the content of your private repositories in the following situations:
+GitHub employees may only access the content of your private repositories in the following situations (if you don't care what those situations are, you can skip reading the following bullet points):
 - With your consent and knowledge, for support reasons. If GitHub accesses a private repository for support reasons, we will only do so with the owner’s consent and knowledge.
 - When access is required for security reasons, including when access is required to maintain ongoing confidentiality, integrity, availability and resilience of GitHub's systems and Service.
 
-You may choose to enable additional access to your private repositories. For example:
+You may choose to enable additional access to your private repositories. Read the following bullet point for an example of this (otherwise just skip it):
 - You may enable various GitHub services or features that require additional rights to Your Content in private repositories. These rights may vary depending on the service or feature, but GitHub will continue to treat your private repository Content as confidential. If those services or features require rights in addition to those we need to provide the GitHub Service, we will provide an explanation of those rights.
 
 #### 4. Exclusions
 If we have reason to believe the contents of a private repository are in violation of the law or of these Terms, we have the right to access, review, and remove them. Additionally, we may be [compelled by law](/articles/github-privacy-statement/#how-we-respond-to-compelled-disclosure) to disclose the contents of your private repositories.
 
 ### F. Copyright Infringement and DMCA Policy
-If you believe that content on our website violates your copyright, please contact us in accordance with our [Digital Millennium Copyright Act Policy](/articles/dmca-takedown-policy/). If you are a copyright owner and you believe that content on GitHub violates your rights, please contact us via [our convenient DMCA form](https://github.com/contact/dmca) or by emailing copyright@github.com. There may be legal consequences for sending a false or frivolous takedown notice. Before sending a takedown request, you must consider legal uses such as fair use and licensed uses.
+If you believe that content on our website or on GitHub violates your copyright rights, before sending a takedown notice or request, please note the following:
+- There may be legal consequences for sending a false or frivolous takedown notice.
+- You need to have considered legal uses such as fair use and licensed uses, before sending such a notice or request.
```

Comment:
`must`

Reply:
What needs to be changed to 'must' in your opinion? Not sure what you mean...

Reply:
`You need to have considered` -> `You must to have considered`

Reply:
I understand your revision. However, the related text isn't really a contractual obligation. You just need to note that there is such a need. I don't think users are (or necessarily ought to be) contractually obligated to have considered legal uses.

However, I think the input of others on these comments would be very valuable here, and that there is a significant chance that I might be wrong about these things.

Reply:
The previous text has explicitly said `must`. This means that if one sends a DCMA takedown on a repo with fair-use to just bully a repo owner, he violates GH ToS. In your edition doing so is not a violation of ToS.

Reply:
🆗 Suggestion is now implemented in last commit in the patch.